packages/tasks/src/model-libraries.ts

```diff
@@ -787,6 +787,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path_extension:"sentis"`,
 	},
+	"sana": {
+		prettyLabel: "Sana",
+		repoName: "Sana",
+		repoUrl: "https://github.com/NVlabs/Sana",
+		countDownloads: `path_extension:"pth" OR path_extension:"json"`,
```

Comment:
.pth file each time the pipeline is instantiated. Is that the case or not? And is the config.json ever loaded from script?

1. If the script only loads the .pth file, then let's count only on .pth:
```suggestion
		countDownloads: `path_extension:"pth"`,
```

2. If the script loads the config file as well, then it's even better and we should count these calls and only these calls. config.json is already tracked by default and that's why some count are already listed on the Hub (see https://huggingface.co/Efficient-Large-Model/Sana_1600M_512px). In that case, no need to add a countDownloads rule.

```suggestion
```


I let you decide what's best for Sana library.

Reply:
Actually, we didn't have a config.json file until yesterday. We want to recover it, so we add pth here. Is this reasonable?

Reply:
ok, makes sense to track pth files then :+1: